core/templates/dev/head/domain/collection/CollectionPlaythroughObjectFactory.js

```diff
@@ -73,9 +75,9 @@ oppia.factory('CollectionPlaythroughObjectFactory', [function() {
   };
 
   CollectionPlaythrough.create = function(
-      nextExplorationIds, completedExplorationIds) {
+      nextExplorationId, completedExplorationIds) {
     return new CollectionPlaythrough(
-      angular.copy(nextExplorationIds), angular.copy(completedExplorationIds));
+      angular.copy(nextExplorationId), angular.copy(completedExplorationIds));
```

Comment:
The first arg is a string so I don't think you need angular.copy() there.

Reply:
Done.